src/pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -174,7 +178,7 @@ def _configure_progress_bar(self, enable_progress_bar: bool = True) -> None:
             )
 
         if enable_progress_bar:
-            progress_bar_callback = TQDMProgressBar()
+            progress_bar_callback = RichProgressBar() if _RICH_AVAILABLE else TQDMProgressBar()
```

Comment:
I'm hesitant of this change for two reasons:

1. Rich pbar is slower than tqdm (there's an issue about this somewhere, cc @akihironitta)
2. I believe CI installs rich, which means that now tqdm is untested. Since it's the most widely used, I find that risky.